apps/cowswap-frontend/src/common/hooks/useSafeMemo.ts

```diff
@@ -2,14 +2,11 @@ import { EffectCallback, useEffect, useMemo } from 'react'
 
 import { CurrencyAmount, NativeCurrency, Percent, Token } from '@uniswap/sdk-core'
 
-import { WrappedTokenInfo } from 'legacy/state/lists/wrappedTokenInfo'
-
 export function useSafeDeps(deps: unknown[]): unknown[] {
   return deps.map((dep) => {
     if (dep instanceof NativeCurrency) return dep.symbol
     if (dep instanceof Token) return dep.address.toLowerCase()
-    if (dep instanceof CurrencyAmount) return dep.toExact() + dep.currency.symbol + dep.currency.isNative
-    if (dep instanceof WrappedTokenInfo) return dep.address
+    if (dep instanceof CurrencyAmount) return dep.toExact() + dep.currency.symbol + dep.currency.chainId
```

Comment:
Hmmm, shouldn't we use the `address.toLowerCase()` here instead of symbol, since symbol is not unique?